src/utils/IO/base_checks.jl

```diff
@@ -5,7 +5,7 @@ function orderedlimits(limits::Union{NamedTuple{(:min, :max), Tuple{Float64, Flo
         @info "'$limitsname' limits defined as nothing"
     else
         if limits.max < limits.min
-            throw(DataFormatError("$limitsname limits not in ascending order"))
+            throw(IS.DataFormatError("$limitsname limits not in ascending order"))
```

Comment:
We re-exported DataFormatError intentionally.  We should discuss.

Reply:
Resolved in https://github.com/NREL/PowerSystems.jl/pull/390/commits/9294f19f8db4db08e2ffd1dc99395402513bbe78.